src/effects.js

```diff
@@ -50,6 +50,7 @@ export function createEffectHandler(
     const helpers = {
       dispatch: references.dispatch,
       getState: () => get(definition.meta.parent, references.getState()),
+      getActions: () => get(definition.meta.parent, _actionCreators),
```

Comment:
@methyl Is `getActions` possibly redundant / what is the use-case for using `getActions` over `actions`?  `getStoreActions` makes sense because otherwise we have no way to access the top-level store's actions, however wouldn't `getActions()` and the first argument of the handler function (`actions`) be identical here? 

With this added, I think there would be three ways to access the local store's actions: 1) via the `actions` argument, 2) `getStoreActions().localStoreName`, and 3) `getActions()`.

I believe that adding this as a helper also makes it inconsistent with `thunk` and `thunkOn` which only expose `getStoreActions`.

Just wanted to point this out and see if I may be missing something here. Thanks for adding this PR, though! I also noticed that `storeState` was `undefined` when playing around with this api. 

cc: @ctrlplusb

Reply:
I agree. Given that the store actions are provided via the first argument for `effectOn`, I do not see the need for a `getActions` helper.

As you said; `getStoreActions` is really handy to expose 👍

Reply:
Hey, not sure as it was quite long time ago, but maybe it's because `actions` can get stale somewhere in between creating effect handler and the handler being called? And `getActions` will always return fresh actionCreators. If you don't think it's possible, let's get rid of it to keep the API simple.

Reply:
Hey @methyl, @ctrlplusb asked me to clean up the conflicts on this branch and based on the discussion with @jmyrland I removed the `getActions` helper as well. Looks like this PR is ready for merging if you're good with this.

Reply:
As long as the test passes (based on the example from #596) - we should be good :+1: